packages/replayer/src/replay.utils.ts

```diff
@@ -209,7 +209,7 @@ export const getSessionDuration: (
     userEventEventLog[userEventEventLog.length - 1]?.timeStamp;
 
   if (maxUserEventTimestamp == null) {
```

Comment:
Is the session duration based on `rrweb` still accurate given we now selectively record rrweb data for sessions?

Reply:
It handles the case of rrweb not being present. I left this in for backwards compatibility -- otherwise you may get differing screenshots for replays of sessions that do have rrweb data compared to the screenshots that used the rrweb duration instead of the last user event timestamp.

I could remove it, but customers may get some one off screenshot diffs (for both virtual time & non-virtual time). If we do remove it probably best in a seperate PR.